src/modules/project/pages1/projectView/views/body/sections/header/Header.tsx

```diff
@@ -160,10 +159,7 @@ export const Header = () => {
             </H1>
 
             <HStack w="full" flexWrap={'wrap'}>
-              <HStack spacing={0.5} maxWidth="full">
-                <LightningAddress name={`${project.name}`} isGeyser />
-                <NpubDisplay npub={project?.keys?.nostrKeys.publicKey.npub} />
-              </HStack>
+              <LightningAddressModal name={`${project.name}`} npub={project?.keys?.nostrKeys.publicKey.npub} />
```

Comment:
Is it necessary to use the `${}` syntax here since it's only the `project.name` that's been passed?

Reply:
I probably did it because the project.name was not string but was Maybe<String> which is string or null or undefined.